### PR TITLE
fix stablediffusion finetune compilation failed

### PR DESCRIPTION
**Pull request by Yancey1989**
fix sd finetune example run failed on main branch.

### CODE REVIEW

File: examples/PyTorch/Train/Dynamo/StableDiffusion/train_dreambooth.py
Comment (yunzhongOvO) [inline examples/PyTorch/Train/Dynamo/StableDiffusion/train_dreambooth.py:1]: No cuda profiler API defined for `nsys`?

**Review by qiuxiafei — DISMISSED** [review]
LGTM~

File: examples/PyTorch/Train/Dynamo/StableDiffusion/train_dreambooth.py
Comment (yunzhongOvO) [inline examples/PyTorch/Train/Dynamo/StableDiffusion/train_dreambooth.py:1]: Seems lack of argument `--enable_torch_compile`.

Comment (Yancey1989) [reply examples/PyTorch/Train/Dynamo/StableDiffusion/train_dreambooth.py:1]: done

Comment (Yancey1989) [reply examples/PyTorch/Train/Dynamo/StableDiffusion/train_dreambooth.py:1]: let's enable `torch.compile` by default.

**Review by qiuxiafei — APPROVED** [review]

**Pull request merged**